types/validator_set.go

```diff
@@ -954,7 +954,7 @@ func (vals *ValidatorSet) VerifyCommit(chainID string, blockID BlockID, stateID
 	blockSignID := commit.CanonicalVoteVerifySignID(chainID, vals.QuorumType, vals.QuorumHash)
 
 	if !vals.ThresholdPublicKey.VerifySignatureDigest(blockSignID, commit.ThresholdBlockSignature) {
-		canonicalVoteBlockSignBytes := commit.StateID.SignBytes(chainID)
+		canonicalVoteBlockSignBytes := commit.CanonicalVoteVerifySignBytes(chainID)
```

Comment:
It seems this change is not related to logging at all.

Reply:
It is related, as `canonicalVoteBlockSignBytes` is only used to generate error message in the next line. And this is a bugfix.